fastlane/lib/fastlane/actions/slather.rb

```diff
@@ -259,12 +261,44 @@ def self.available_options
                                        env_name: "FL_SLATHER_BINARY_BASENAME",
                                        description: "Basename of the binary file, this should match the name of your bundle excluding its extension (i.e. YourApp [for YourApp.app bundle])",
                                        is_string: false,
-                                       default_value: false),
+                                       optional: true,
+                                       conflicting_options: [:binary_basenames],
+                                       verify_block: proc do |value|
+                                         if value.kind_of?(Array)
+                                           UI.deprecated(":binary_basename will only accept a String in the future. Please migrate over to using :binary_basenames")
+                                         end
+                                       end,
+                                       conflict_block: proc do |value|
+                                         UI.user_error!("You can't use 'binary_basename' and 'binary_basenames' options in one run")
+                                       end),
+          FastlaneCore::ConfigItem.new(key: :binary_basenames,
```

Comment:
Do we support passing in a flag multiple times for a list of parameters? that would feel more unixy than adding a new switch.